api/pkg/apis/v1alpha1/managers/configs/configs-manager.go

```diff
@@ -58,7 +58,8 @@ func (s *ConfigsManager) Init(context *contexts.VendorContext, cfg managers.Mana
 	return nil
 }
 func (s *ConfigsManager) Get(object string, field string, overlays []string, localContext interface{}) (interface{}, error) {
-	if strings.Index(object, ":") > 0 {
+	log.Infof(" M (Config): Get %v, %d", object, len(s.ConfigProviders))
```

Comment:
Same here, Configs manager doesn't handle versioning

Reply:
I think you misunderstand this part. It is to support config reference: `${{$config(‘my-config:v3’, ‘my-field’)}}
`.